.travis.yml

```diff
@@ -1,12 +1,15 @@
 language: python
-node_js: "8.4.0"
+node_js: "8"
 
 cache:
-  directories:
-    - node_modules
+  - pip
+  - yarn
 
 env:
-  - DISPLAY=':99.0'
+  global:
+    - DISPLAY=':99.0'
+    - YARN_VERSION='0.24.5'
+    - MC_COMMIT='abd295766dfe'
```

Comment:
Are you now able to bump this m-c changeset easily/frequently?

Reply:
Yep. It's fixed so that we always get a good artifact build though